### ISSUE TITLE
abort sortleave event

### ISSUE THREAD

**Issue by giacomomasseron** [lukasoppermann/html5sortable#790]
Is there the chance to abort the sortleave event and avoid an item from being removed from the list?

**Comment by lukasoppermann** [comment]
Hey, there is no specific functionality for this. But you should be able to add it to the sortleave event handler.

Did you try a simple `event.preventDefault()`?

**Comment by firanto** [comment]
> Hey, there is no specific functionality for this. But you should be able to add it to the sortleave event handler.
> 
> Did you try a simple `event.preventDefault()`?

I've tried to do it as you've said (calling `event.preventDefault()` on `sortleave` event). The dragged item is still lost forever (it didn't back to its original position before it is dragged).

**Comment by lukasoppermann** [comment]
Hmm, in this case it is really missing. Would you be able to send a PR that adds this functionality?

**Comment by ac615223s5** [comment]
how do you prevent the item from being removed in your demo?

**Comment by lukasoppermann** [comment]
@zzaria what specifically are you referring to?

**Comment by ac615223s5** [comment]
https://lukasoppermann.github.io/html5sortable/index.html in the first sortable list, the element doesn't disappear when dragged off, it just returns to its original postion when the mouse is released
![image](https://user-images.githubusercontent.com/33037196/160022142-8a06d558-471a-45dd-944e-5a3ff6ca6776.png)

**Comment by zhixXiang** [comment]
any update?

**Comment by lukasoppermann** [comment]
No. But I am very happy to accept a PR if you are able to send one.
